chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -989,6 +1003,9 @@ public synchronized void loadChunks(TaskTracker taskTracker, World world, Collec
                   int octNode = currentBlock;
                   Block block = palette.get(currentBlock);
 
+                  if(block.useBiomeTint())
```

Comment:
Maybe?

```suggestion
                  if(block.isBiomeDependant())
```

Reply:
Why not, wasn't too sure of the name myself

Reply:
How about `isBiomeTinted`? Biome dependant sounds like a block "_design_" can change depending on the biome.

Reply:
How about `canHaveBiomeTint`?